cmd/podman/system/service_abi.go

```diff
@@ -55,3 +60,23 @@ func restService(opts entities.ServiceOptions, flags *pflag.FlagSet, cfg *entiti
 	}
 	return err
 }
+
+// startWatcher starts a new SIGHUP go routine for the current config.
+func startWatcher(rt *libpod.Runtime) {
+	// Setup the signal notifier
+	ch := make(chan os.Signal, 1)
+	signal.Notify(ch, utils.Hup)
+
+	go func() {
+		for {
+			// Block until the signal is received
+			<-ch
```

Comment:
Could we add a debug message here as well? This could come in handy for support.